src/1_utils.js

```diff
@@ -263,3 +263,20 @@ utils.base64encode = function(input) {
 	}
 	return output;
 };
+
+/**
+ * Extract the deeplink path from urls in the forms of:
+ * - "protocol://domain.name/deeplink.path
+ * - "domain.name/deeplink.path"
```

Comment:
it's just a place holder name, unrelated to our internal concept of `$deepview_path`. This function is just a string parser.

Serious, what is the name of the part of the url after the domain name? Like, for example, the url is https://github.com/BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/pull/212#discussion_r42532884 , what words do we use to describe the `BranchMetrics/Smart-App-Banner-Deep-Linking-Web-SDK/pull/212#discussion_r42532884` part?